tidegates/testing/helpers.py

```diff
@@ -70,7 +68,7 @@ class NoseWrapper(nptest.Tester):
 
 
     def test(self, label='fast', verbose=1, with_id=True, exe=True,
-             doctests=False, coverage=False, packageinfo=True, extra_argv=[],
+             doctests=False, coverage=False, packageinfo=True, extra_argv=None,
```

Comment:
The numpy nose wrapper requires this to be a list. If you're going to change it to `None`, you need to check for that and set it to an empty list before it gets used.